### PR TITLE
Product-page

### PR DESCRIPTION
**Pull request by Shailseen**
image
image

### CODE REVIEW

File: pages/homepage/product-page.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+.badge-card{
+    background-color: var(--white-color);
+    color: var(--dark-secondary-color);
+    margin-left: 9rem;
+}
+.badge-card:hover{
+    color: var(--red-color);
+}
+.card-vertical:hover{
```
Comment (ckahub) [inline pages/homepage/product-page.css:91]: use rem everywhere

File: pages/homepage/product-page.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    margin: 0;
+}
+.border-bottom{
+    width: 100%;
+    border: 1px solid var(--dark-secondary-color);
+    border-top: 0;
+    border-left: 0;
+    border-right: 0;
+}
+.box-shadow1{
```
Comment (ckahub) [inline pages/homepage/product-page.css:80]: use rem

**Review by ckahub — COMMENTED** [review]
lgtm

**Review by ishitakeshawani — APPROVED** [review]
LGTM

File: pages/homepage/product-page.css (hunk 2 above)
Comment (Shailseen) [reply pages/homepage/product-page.css:80]: okay.

**Pull request merged**